examples/platform/efr32/MemMonitoring.cpp

```diff
@@ -22,6 +22,9 @@
 #include "FreeRTOS.h"
 #include <platform/CHIPDeviceLayer.h>
 
+#define BLE_STACK_TASK_NAME "Bluetooth s"
```

Comment:
Why is the name not more descriptive? Bluetooth is very lon .. why not BLE_STACK / BLE LINK or similar, to make them a bit clearer?

Reply:
The names have been changed to their complete version. A define was added to preserver the Task Name size to 10 bytes when not using heap monitoring.